RELEASE_NOTES.md

```diff
@@ -7,6 +7,29 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current major release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.15.X (TBD)
+
+* [Upgrade to PostgreSQL 9.6](https://github.com/chef/chef-server/pull/1310),
+  Chef Server now uses the latest stable version of the 9.6 series (9.6.3). Upgrades of existing
+  installations are done automatically, but creating backups is advised.
+  Note that the `checkpoint_segments` configuration setting is gone, so if you before have set
```

Comment:
Yeah you're right. 🙄 (Just kidding. You simple _are_ right there, and I'll add this 😉)